geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/filters/CqlTransformFilterTest.scala

```diff
@@ -0,0 +1,78 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
```

Comment:
2019?

Reply:
the copyright templates for 2.1 and 2.2 weren't updated to 2018 - we should probably go back and do that.